policyengine_us/parameters/gov/states/mi/tax/income/deductions/retirement_benefits/expanded/birth_year.yaml

```diff
@@ -0,0 +1,27 @@
+description: Michigan allows for an expanded retirement and pension benefits deduction if this is true, based on their year of birth.
+brackets:
+  - threshold:
+      2023-01-01: 0
+    amount:
+      2023-01-01: false
+  - threshold:
+      2023-01-01: 1956
+    amount:
+      2023-01-01: true
+  - threshold:
+      2023-01-01: 1959
+    amount:
+      2023-01-01: false
+metadata:
+  type: single_amount
+  amount_unit: bool
+  threshold_unit: year
+  period: year  
+  label: Michigan expanded retirement and pension benefits birth year range
+  reference:
+    - title: Michigan Legal Code Section 206.30 - (10)
+      href: http://legislature.mi.gov/doc.aspx?mcl-206-30
```

Comment:
10(a) says:
>after 1945 and before 1959

where does 1956 come from?

also can you please future date the policy based on the law? it changes through 2026

Reply:
1956 appears in tax form instruction: https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2023/2023-IIT-Forms/BOOK_MI-1040.pdf#page=20

Reply:
Is this resolved? Could you explain if so?

Reply:
Because we noticed Michigan 2023 expanded deduction was also applied to tier 2 & 3 standard deductions, which include birth year 1946-1956. 

> If the older of you or your spouse (if filing a joint return) was born during the period January 1, 1946 through January 1, 1957, you
are eligible to claim the larger of your pension and retirements benefits subtraction calculated on line 6 of this worksheet or the
amount calculated on Worksheet 2: Tier 2 or Tier 3 Michigan Standard Deduction.
If you were instructed to compare this worksheet with Worksheet 3.2, you are eligible to claim the larger of your pension and
retirements benefits subtraction calculated on line 6 of this worksheet or the amount calculated on Worksheet 3.2. 

https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2023/2023-IIT-Forms/BOOK_MI-1040.pdf#page=25

Reply:
So now the birth year for expanded deduction are after 1945 and before 1959, which are exactly the same as the legal code.